hack/make-rules/test.sh

```diff
@@ -23,6 +23,7 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 
 kube::golang::setup_env
 kube::golang::setup_gomaxprocs
+kube::util::ensure-temp-dir
```

Comment:
IIUIC this creates a temp dir and stores the path in `KUBE_TEMP` but I don't see we use that var later, am I missing something?

Reply:
I can remove this again. It's a leftover from a previous iteration where KUBE_TEMP was used.